.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+# https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuring-dependabot-version-updates
+
+version: 2
+updates:
+  - package-ecosystem: 'npm'
+    directory: '/'
+    schedule:
+      interval: 'weekly'
+    open-pull-requests-limit: 10
```

Comment:
Limit may not even be necessary. We have seen that if you change dependabot.yml, Dependabot closes PRs that do not fit the new config. So should we ever get hundreds of PRs we don't want, that is an easy fix by changing dependabot.yml.

Reply:
Thx! I have removed this line